### DISCUSSION TITLE
stats.tsv file - What is MS2.Mass.Instability and Normalisation.Instability columns?

### DISCUSSION THREAD

**Discussion by frederikhvh** [vdemichev/DiaNN#932] [Q&A]
Hi, 
I've been a DIANN user for quite some time now. 
Most of the columns of the stats file output are sensible to me from their names alone, however, the two columns "MS2.Mass.Instability" and "Normalisation.Instability" are puzzling. 
Can anybody explain what these columns contain? Thanks a lot!

**Comment by vdemichev** [comment]
These reflect how mass calibration in the acquisition varies with time (retention time), and respectively how the signal intensity varies with time in relation to other runs. The latter will capture various effects due to relative depletion of hydrophobic or hydrophilic peptides in a paerituclar run or spray instability.

**Reply by frederikhvh** [reply]
That makes sense, thanks for the answer
